### ISSUE TITLE
../scripts/Source folder is empty for apps executed from MO

### ISSUE THREAD

**Issue by zax-ftw** [TanninOne/modorganizer#350]
So for example: I managed to setup Notepad++ to run and compile scripts via MO. Everything is working as intended, .pex files added by mods from MO are visible (when vieving them from say Notepad++ File/Open, or any other app opened from MO), but "Source" folder contains only vanilla content.

**Comment by GrantSP** [comment]
When you examine the files in MO's 'Data' tab, are they visible there?
The files provided by your set of mods that are evident in the "VFS" should be visible to any tool that is run from MO.

**Comment by zax-ftw** [comment]
Yep, both .pex ('scripts' folder) and .psc ('Source' folder) are visible under 'Data' in MO, but when I open it in Notepad++ for example, I see all MO virtual folder structure, I see .pex in 'scripts' folder, but 'Source' folder contains only vanilla content. 

Basically during the same 'Notepad++' session (executed via MO), I see mods loaded in 'scripts', but not in 'Source' folder.

I checked it multiple times already with hope that I do something wrong, but no matter what I do, and how much 'sense' it makes or doesn't, it didn't fix itself. :)

**Comment by zax-ftw** [comment]
I tested it more and it seems that no other apps executed via MO (Nifskope etc, and even windows system32/Notepad.exe) can see non-vanilla files inside 'Source' folder (I pasted .nif and files of other formats inside 'source' scripts folder to test it). 

This is what actually happens:
- excluding this specific 'scripts/source' directory, I can *browse* whole virtual folders structure from any app executed via MO and *see* any files (.nifs in 'meshes', .dds in 'textures', .pex in 'scripts)
- I can't *see* anything added via MO in 'Source' scripts folder
- its not app specific issue, because neither of apps could see files
- its not .psc specific, because neither .nifs nor files of other formats were visible either

Here's something special:
-when I try to save file with the same name as the file that SHOULD exist (but is hidden?) in 'source' scripts folder I'm getting "FILE ALREADY EXISTS" pop-up message. 
So its here, but not displayed from File->Open window(?)

**Comment by GrantSP** [comment]
I just tried it and you are correct but it isn't only the 'Source' folder, other folders exhibit this behaviour.
If you navigate to any mod installed via MO there should exist in every top level a 'meta.ini' file, this file too will not show in the open file dialogues.
Not sure what is going on, whether it is by design or error?

**Comment by zax-ftw** [comment]
@GrantSP, Hmm, I can see 'meta.ini' in my 'Data' folder. I don't think you should be navigating to individual mods folders, because they arent supposed to be accessed from outside.

Anyway, **I found workaround for this issue.** Files in 'source' folder are indded here, but are hidden in 'open file' dialogue and can be accessed by writing their absolute path (with filename):

- D:\Games\Steam\SteamApps\common\Skyrim\Data\Scripts\Source\MyScrpt.psc

That way the file will open in external apps, even though its hidden in their File->Open window.


**Still, it would be awesome to hear Tannin's comment about this issue.**

**Comment by zax-ftw** [comment]
@TanninOne, You probably don't care about 32bit version anymore, but heres what I found:
- some files from real 'Data\Scripts\Source' folder are hidden for apps launched from MO!

I compared original 'source' folder files that are hidden with the ones that arent, but everything is same, same attributes and all. It really doesn't make sense.

**Comment by ThosRTanner** [comment]
"Is working on 64 bit version" does not mean "doesn't care about 32 bit version". There's plenty of games still being played that require 32 bit

**Comment by zax-ftw** [comment]
Hmm, I somewhat imagined 64 bit version being automaticly backwards compatible with 32 everything? Forgive my ignorance!

**Comment by TanninOne** [comment]
It will be compatible to 32-bit applications but not to the bugs! ;)
 Its a complete rewrite so there is no reason to assume this problem will still exist.
This is why i announced i would only fix critical bugs a while ago.
